dataflux_pytorch/benchmark/checkpointing/multinode/README.md

```diff
@@ -0,0 +1,37 @@
+# Benchmarking PyTorch Lightning Checkpoints with Google Cloud Storage
+
+This benchmarking script will allow you to run and benchmark the performance of the PyTorch Lightning Checkpoint save/load function. The multinode benchmarking script does require a GPU. The script runs the `WikiText2` PyTorch Lightning demo code with some modifications.
```

Comment:
Can you add some more details here or somewhere in the readme with information about the checkpointing strategy used here and why it's valid to run it on just one machine even though this is called "multinode"?

Reply:
Done